src/main/java/frc/robot/RobotContainer.java

```diff
@@ -172,6 +180,8 @@ private void configureBindings() {
 
         handoffTrigger.and(() -> !tuneSetpointSubscriber.getBoolean(false)).whileTrue(handoffCommand);
 
+        reZeroClimbTrigger.onTrue(new ZeroClimbersCommand(climb));
+
```

Comment:
Hmm, we'll need to do some testing to see if this actually triggers when the robot first boots.

I'm not 100% certain how the `.onTrue()` decorator works. If it's only on a rising edge, and the field starts out at `true`, then there's a possibility it won't trigger.

If you want to be save, I'd add another RobotModeTriggers trigger, like with the intake.

Something like
```java
(RobotModeTriggers.autonomous().or(RobotModeTriggers.teleop()))
                .and(() -> !climb.bothZeroed())
                .onTrue(zeroClimbCommand);
````